docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -713,7 +713,8 @@ end::payloads[]
 
 tag::pipeline[]
 `pipeline`::
-(Optional, string) ID of the pipeline to use to preprocess incoming documents.
+(Optional, string) ID of the pipeline to use to preprocess incoming documents. The special pipeline
+name `_none` indicates no ingest pipeline will run, other than the final pipeline if it exists.
```

Comment:
```suggestion
(Optional, string) ID of the pipeline to use to preprocess incoming documents. If the index has a default ingest pipeline specified, then setting the value to `_none` disables the default ingest pipeline for this request. If a final pipeline is configured it will always run.
```

What do you think? Perhaps we could link to docs for both default pipelines and final pipelines?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/8.6/index-modules.html#index-default-pipeline

and

https://www.elastic.co/guide/en/elasticsearch/reference/8.6/index-modules.html#index-final-pipeline

Might also be a good idea to double check those sections. For some reason it states that _none will keep the final pipeline from running. Unless it means setting that on the index settings?

Reply:
Yeah that is saying that if you set the final pipeline to `_none` then it will keep the final pipeline from running. I'll make that a little more clear while I'm in here. Your wording sounds good to me -- I'll change it to something more like that.